Modules/DataCollection/classes/Fields/Base/class.ilDclBaseRecordFieldModel.php

```diff
@@ -28,7 +28,7 @@
  */
 class ilDclBaseRecordFieldModel
 {
-    protected int $id = 0;
```

Comment:
I'm not so sure that this will break somewhere down the line. So I would like to avoid it. Can you point out at which places this is needed?